wsapi.go

```diff
@@ -379,6 +379,23 @@ func (s *Session) UpdateStatus(idle int, game string) (err error) {
 	return s.UpdateStreamingStatus(idle, game, "")
 }
 
+//UpdateListeningStatus is used to set the user to "Listening to..."
+//If game!="" then set to what user is listening to
+//Else, set user to active and no game.
+func (s *Session) UpdateListeningStatus(idle int, game string) (err error) {
```

Comment:
Let's make this:

UpdateListeningStatus(game)

Idle seems kinda useless and people can do it with Complex.